src/compiler/types.ts

```diff
@@ -759,6 +759,7 @@ namespace ts {
         kind: SyntaxKind.TypeParameter;
         parent: DeclarationWithTypeParameterChildren | InferTypeNode;
         name: Identifier;
+        // Note: Consider calling `getEffectiveConstraintOfTypeParameter`

```

Comment:
Hm. We should have these notes as jsdoc for lots of things now. Something for a separate PR though.